src/hooks/v2/contractReader/ProjectCurrentFundingCycle.ts

```diff
@@ -5,11 +5,15 @@ import useV2ContractReader from './V2ContractReader'
 
 export default function useProjectCurrentFundingCycle({
   projectId,
+  useDeprecatedContract,
 }: {
   projectId?: number
+  useDeprecatedContract?: boolean
```

Comment:
where necessary, pass in a flag to these hooks so we can use the new or deprecated (old) contracts